### PR TITLE
Update to SDK 1.0.0-rc.8

### PR DESCRIPTION
**Pull request by smallhive**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -68,12 +71,12 @@ require (
...
 	github.com/urfave/cli v1.22.5 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
-	go.uber.org/multierr v1.8.0 // indirect
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f // indirect
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
-	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211 // indirect
-	golang.org/x/text v0.3.7 // indirect
+	go.uber.org/multierr v1.9.0 // indirect
+	golang.org/x/net v0.3.0 // indirect
```
Comment (roman-khimov) [inline go.mod:75]: This one is still outdated, unfortunately (CVE-2022-41723). Please, upgrade to 0.7.0+ in another commit.

**Comment by roman-khimov** [comment]
BTW, #774 also mentions nspcc-dev/neofs-sdk-go#231 that should be fixed there.

File: api/layer/layer.go [hunk 2]
```diff
@@ -322,7 +321,9 @@ func (n *layer) Owner(ctx context.Context) user.ID {
 	}
 
 	var ownerID user.ID
-	user.IDFromKey(&ownerID, (ecdsa.PublicKey)(*n.EphemeralKey()))
+	if err := user.IDFromKey(&ownerID, n.EphemeralKey().Bytes()); err != nil {
+		n.log.Error("if from key", zap.Error(err))
```
[the remark this answers is not in the document]
Comment (smallhive) [reply api/layer/layer.go]: Put panic

File: go.mod (hunk 1 above)
Comment (smallhive) [reply go.mod]: updated

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**